core/trino-main/src/test/java/io/trino/sql/query/TestRowFilter.java

```diff
@@ -554,4 +596,33 @@ public void testRowFilterOnHiddenColumn()
                 .skippingTypesCheck()
                 .matches("SELECT BIGINT '25'");
     }
+
+    @Test
+    public void testRowFilterOnOptionalColumn()
+    {
+        accessControlMissingColumns.reset();
```

Comment:
How about extracting this to `@BeforeMethod` (please do it as separate commit).?
FYI Please follow: https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#git-merge-strategy